python/src/otel/otel_sdk/otel-instrument

```diff
@@ -89,14 +89,9 @@ export PYTHONPATH="$LAMBDA_RUNTIME_DIR:$PYTHONPATH";
 
 # - Set the trace exporter
 
-if [ -z "${OTEL_TRACES_EXPORTER}" ]; then
-    export OTEL_TRACES_EXPORTER=otlp_proto_http;
-fi
-
-# - Set the metrics exporter
-
-if [ -z "${OTEL_METRICS_EXPORTER}" ]; then
-    export OTEL_METRICS_EXPORTER=none;
+# - If OTEL_EXPORTER_OTLP_TRACES_PROTOCOL is not set by user, the default exporting protocol for traces is http/protobuf.
+if [ -z "${OTEL_EXPORTER_OTLP_TRACES_PROTOCOL}" ]; then
+    export OTEL_EXPORTER_OTLP_TRACES_PROTOCOL=http/protobuf
```

Comment:
```suggestion
# - If OTEL_EXPORTER_OTLP_PROTOCOL is not set by user, the default exporting protocol is http/protobuf.
if [ -z "${OTEL_EXPORTER_OTLP_PROTOCOL}" ]; then
    export OTEL_EXPORTER_OTLP_PROTOCOL=http/protobuf
```

I think we should set the signal-agnostic protocol variable if not set and leave signal-specific settings to the user.  [The spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#specify-protocol) says this should be the default in the absence of explicit configuration.

Reply:
Would this not change the exporting protocol for metrics too?

Reply:
I don't think we should be overriding the default at all, so would argue leaving it to only traces is better. But this means adding back `export OTEL_METRICS_EXPORTER=none;` in this PR.

Reply:
> But this means adding back `export OTEL_METRICS_EXPORTER=none;` in this PR.

Why?  This seems to be clearly wrong and the need to explicitly set a configuration option that is supposed to be the default has been a point of confusion for a number of users.

Reply:
It is technically a breaking change. I think both should be using the defaults. So a separate PR where both exports are removed and the fact it is a breaking change can be debated.

Reply:
Okay, I will open up another PR to discuss this breaking change.